docs/src/main/asciidoc/maven-project.adoc

```diff
@@ -255,6 +255,14 @@ components. Those properties might be used by (for example) built-in or custom S
 * `incrementalContractTests`: When enabled, tests are created only when contracts have changed since last build. Defaults to `true`.
 * `incrementalContractStubs`: When enabled, stubs are created only when contracts have changed since last build. Defaults to `true`.
 * `incrementalContractStubsJar`: When enabled, stubs jar is created only when stubs have changed since last build. Defaults to `true`.
+*`httpPort` : HTTP port for the WireMock server that serves stubs. Currently `spring.cloud.contract.verifier.http.port` property works only when serving stubs from the directory. Otherwise, when providing stubs id, port have to be included in the id string.
```

Comment:
Change the second sentence to "Otherwise, when providing stubs IDs, the ports have to be included in the ID string."